src/languages/fi.json

```diff
@@ -1,16 +1,21 @@
 {
-  "filterpanel.heading": "Liikenteenvalvontatyökalu",
-  "filterpanel.reset": "Palauta",
+  "filterpanel.heading": "Transitlog",
+  "filterpanel.reset": "Pyyhki asetukset",
```

Comment:
This would be "palauta asetukset", Pyyhi means clear, but that's not what happens.